packages/reco/interface/SRecEvent.h

```diff
@@ -343,8 +343,8 @@ class SRecDimuon: public SQDimuon
     //Get the total momentum of the virtual photon
     TLorentzVector getVPhoton() { return p_pos + p_neg; }
 
-    //Calculate the kinematic vairables
-    void calcVariables();
+    //Calculate the kinematic vairables, 0 = vertex, 1 = target, 2 = dump
```

Comment:
Can you replace `//` with `///` so that the comment gets visible on the Doxygen page?

Reply:
Good advice, changed the comment style in another commit